internal/rickandmorty/transport/http.go

```
@@ -80,3 +86,36 @@ func HTTPGetCharactersStoredFromAPI(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	writeFormattedResponse(w, characterList, outputFormat)
 }
+
+func HTTPReadConcurrent(w http.ResponseWriter, r *http.Request) {
+	file, _, err := r.FormFile(formFileCSV)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+	defer file.Close()
+
+	outputFormat := r.URL.Query().Get(queryOutputFormat)
+	typee := r.URL.Query().Get(queryType)
+	items, _ := strconv.Atoi(r.URL.Query().Get(queryItems))
```

As Cesar mentioned, ignoring errors is not a good practice.

In these cases that you are trying to convert a string to an integer the variable is going to be assigned a zero; handling the error would help you catch these kinds of scenarios.

If you want to ignore the error, another approach would be to validate these values are distinct than zero, and if so, assign them a default value.